core/src/main/scala/org/apache/spark/sql/execution/columnar/impl/StoreCallbacksImpl.scala

```diff
@@ -137,6 +137,16 @@ object StoreCallbacksImpl extends StoreCallbacks with Logging with Serializable
   override def getHashCodeSnappy(dvds: scala.Array[Object]): Int = {
     partioner.hashValue(dvds)
   }
+
+  override def haveRegisteredExternalStore(tableName: String): lang.Boolean = {
+    // TODO -  remove below that deals with default schema and all
```

Comment:
Why java.lang.Boolean object? This can't return null so just Boolean should be good.